StarCellBio/settings.py

```diff
@@ -195,4 +193,46 @@
     }
 }
 
-AUTH_USER_MODEL='auth.User'
+AUTH_USER_MODEL = 'auth.User'
+
+
+# Override settings with untracked YAML config
+if os.path.isfile(YAML_CONFIG):
+    with open(YAML_CONFIG) as f:
+        y = yaml.load(f)
+        if y is not None:
+            globals().update(y)
+
+# Override settings with SCB_ environment variables
+scb_env_overrides = {
+    key[4:]: value
+    for key, value in os.environ.iteritems()
+    if key.startswith("SCB_")
```

Comment:
This is a bit terse for me as a dictionary comprehension, and it arbitrarily makes it python 2.7 only (and the servers running this are 2.6).